entwatch/ze_2049_v1_1.cfg

```diff
@@ -0,0 +1,78 @@
+"entities"
+{
+    "0"
+    {
+        "name"              "Slow Ball"
+        "shortname"         "Slow"
+        "color"             "{olive}"
+        "buttonclass"       "func_button"
+        "filtername"        "player_human_slowball"
+        "blockpickup"       "false"
+        "allowtransfer"     "true"
+        "forcedrop"         "true"
+        "chat"              "true"
+        "hud"               "true"
+        "hammerid"          "812208"
+        "mode"              "1"
+        "maxuses"           "0"
+        "cooldown"          "0"
+        "pt_spawner"        "human_slowball_itemtemp"
+    }
```

Comment:
Seems like this missed conversion from DZ format? hasfiltername & maxmount are missing & pt_spawner is included.